server/main.go

```diff
@@ -4,20 +4,20 @@ import (
 	"flag"
 
 	glog "github.com/golang/glog"
+	"github.com/otrego/clamshell/server/config"
 	"github.com/otrego/clamshell/server/grpc"
 )
 
-var (
-	port = flag.Int("port", 8080, "The server port")
-)
-
 func main() {
-	flag.Set("alsologtostderr", "true")
+	flag.Set("logtostderr", "true")
```

Comment:
What does this "logtostderr" vs. "alsologtostderr" change do?

Reply:
By default, glog logs to a file. alsologtostderr logs to both a file and to stderr.

But you might as well just *not* log to the file and log to stderr. You can always redirect the logs to a file if you want.